### PR TITLE
[Handbook] added code snippets

### LINKED ISSUE #2137: [UI][Handbook] Use clipboard
**Current Behavior**
Currently all the code snippets in the handbook are in quotes

**Desired Situation**
Use the clipboard component for all the commands.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabh1908@gmail.com>

**Description**
Added code snippets to [Contribution](https://layer5.io/community/handbook/contribution) page of handbook.

This PR fixes #2137 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Community/Handbook/contributing.js [hunk 1]
```diff
@@ -13,43 +14,57 @@ const contributingGuide= () => {
...
+             For example,<Code codeString="git checkout -b feature" />  (feature being a branch name)</span></li>
             </ul>
+
             <h3>Make the necessary changes to your file.</h3>
             <ul>
               <li><span>Now, go to town hacking away and make whatever changes you want to.</span></li>
-              <li><span>To add the changes you have made to your branch, use the git add file.md command.</span></li>
-              <li><span> If you add multiple file changes to the branch, you simply use a git add . command.</span></li>
+              <li><span>To add the changes you have made to your branch, use: 
+              <Code codeString="git add file.md " /></span></li>
```
Comment (warunicorn19) [inline src/sections/Community/Handbook/contributing.js:53]:
```suggestion
              <Code codeString="git add <file> " /></span></li>
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit bafda05b879ff9e7e5861ad937bc6b2baf13d932 at: https://612d31c2f74987228e7dd08f--layer5ng.netlify.app

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@debo19  could you please reduce the size of the box and reduce the distance between the text and the code snippet just a little

Screenshot 2021-08-31 at 1 07 49 AM

**Comment by warunicorn19** [comment]
@leecalcote could you please re-run the build, i can't seem to find the option to, netlify didn't seem to work.

**Comment by leecalcote** [comment]
Hmm. Hmm.

**Comment by warunicorn19** [comment]
@debo19 @leecalcote maybe we could try adding a dummy commit to this PR and let's see if it goes off. Seemed to be working fine 2mins ago.

File: src/sections/Community/Handbook/Handbook.style.js [hunk 2]
```diff
@@ -73,12 +62,21 @@ h2 h3{
         }
     }
     
+     .codes{
+      width:75%
+      margin-top:-2rem;
+    }
+
```
Comment (leecalcote) [inline src/sections/Community/Handbook/Handbook.style.js:69]:
```suggestion
```

Adding a simple commit to kick the build process....

**Comment by l5io** [comment]
🚀 Preview for commit 8e698212fa28ca38b54e896847c3ecabd2fcc0e0 at: https://612d970f18078f45556f8684--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉, thanks @debo19

**Review by leecalcote — APPROVED** [review]
Looks great.

**Review by Tanuj22 — APPROVED** [review]

**Pull request merged**